### PR TITLE
Tweak(Scoreboard): Match the new FiveM UI

### PR DESCRIPTION
**Pull request by itschip**

### CODE REVIEW

**Comment by itschip** [comment]
Scoreboard

**Comment by itschip** [comment]
New design, agian,

File: resources/[system]/scoreboard/html/style.css [hunk 1]
```diff
@@ -9,10 +13,17 @@ html {
...
 	margin-right: auto;
-	background-color: #fff;
+	background-color: #2d3436;
     box-shadow: 0px 2px 5px 0px rgba(0, 0, 0, 0.16), 0px 2px 10px 0px rgba(0, 0, 0, 0.12);
-    color: rgba(255, 255, 255, 0.9);
+    color: #fff;
+}
+
+
+#playerInfo {
```
Comment (technetium-cfx) [inline resources/[system]/scoreboard/html/style.css:24]: IDs are for unique elements only, use class names for repeating elements.

File: resources/[system]/scoreboard/scoreboard.lua [hunk 2]
```diff
@@ -13,7 +13,7 @@ Citizen.CreateThread(function()
                     local wantedLevel = GetPlayerWantedLevel(i)
                     r, g, b = GetPlayerRgbColour(i)
                     table.insert(players, 
-                    '<tr style=\"color: rgb(' .. r .. ', ' .. g .. ', ' .. b .. ')\"><td>' .. GetPlayerServerId(i) .. '</td><td>' .. sanitize(GetPlayerName(i)) .. '</td><td>' .. (wantedLevel and wantedLevel or tostring(0)) .. '</td></tr>'
+                    '<tr style=\"color: #fff;\"><td id="playerInfo">' .. GetPlayerServerId(i) .. '</td><td>' .. sanitize(GetPlayerName(i)) .. '</td><td>' .. (wantedLevel and wantedLevel or tostring(0)) .. '</td></tr>'
```
Comment (technetium-cfx) [inline resources/[system]/scoreboard/scoreboard.lua:16]: Please keep the wanted level color in some way, matching the design. Can't just remove functionality 😄

**Comment by jaymo1011** [comment]
This might be somewhat out-of-scope for the base resource but considering the widespread adoption OneSync and >32 player counts, would it be possible to allow more than one column of players with this same design?
If yes, it would also be neat if the number of columns was based on the `sv_maxClients` convar.

Just a suggestion though, what you've already done looks great! 😄

File: resources/[system]/scoreboard/html/scoreboard.html [hunk 3]
```diff
@@ -3,6 +3,7 @@
 		<script src="nui://game/ui/jquery.js" type="text/javascript"></script>
 		<script src="listener.js" type="text/javascript"></script>
 		<link rel="stylesheet" href="nui://scoreboard/html/reset.css">
+		<link href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;500&display=swap" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (itschip) [reply resources/[system]/scoreboard/html/scoreboard.html:6]: Sure can do. I was just being a bit rushed and I didn't see that I added the import inside * {}

**Comment by itschip** [comment]
@jaymo1011 sure! I could always just make it bigger depending on the player count, or keep it in one size.

**Comment by itschip** [comment]
Nothing in the design has changed other than a wider size and some clean up.

**Pull request closed without merging**